R/reporter-tap.R

```diff
@@ -29,20 +29,21 @@ TapReporter <- R6::R6Class("TapReporter", inherit = Reporter,
       if (!self$has_tests)
         return()
 
-      cat("1..", self$n, '\n', sep = '')
+      self$cat_line("1..", self$n)
       for (i in 1:self$n) {
         if (!is.na(self$contexts[i])) {
-          cat("# Context", self$contexts[i], "\n")
+          self$cat_line("# Context ", self$contexts[i], " ")
```

Comment:
@dkesh: I'm reworking output for the TAP reporter. Does it require trailing whitespace, or would it work without them?